lib/python/qmk/cli/new/keyboard.py

```diff
@@ -0,0 +1,98 @@
+"""This script automates the generation of a new keyboard directory.
+"""
+import os
+import shutil
+
+from distutils.dir_util import copy_tree
+from datetime import datetime
+from milc import cli
```

Comment:
Please group these per pep8. Moving the blank line down 2 lines should do it.

Reply:
According to pep8, there should be a blank line between each group of imports.

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> 
> Imports should be grouped in the following order:
> 
> - Standard library imports.
> - Related third party imports.
> - Local application/library specific imports.
> 
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Do you still want me to group them?

Reply:
Exactly, that's why it should be :

```
import os
import shutil
from distutils.dir_util import copy_tree
from datetime import datetime

from milc import cli
```

Reply:
🤦‍♂

Reply:
Fixed, sorry.